### ISSUE TITLE
Manpage causes messages "UNSUPP: ignoring macro in table" when viewed in macOS

### ISSUE THREAD

**Issue by alebcay** [leo-arch/CliFM#183]
**Describe the bug**
When viewing the manpage for `clifm`, some warning/error messages are printed (not in the manpage viewer, but visible once the pager for `man` exits).

**To Reproduce**
1. Clone this repo.
2. Attempt to view the manpage with `man`: `man ./misc/manpage`
3. The manpage will be displayed, without any obvious errors/issues. Press `q` to exit the pager.
4. `mandoc` has printed a bunch of warnings/errors about `UNSUPP: ignoring macro in table`

```
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1951:2: UNSUPP: ignoring macro in table: BR bn
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1954:2: UNSUPP: ignoring macro in table: BR bcp
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1957:2: UNSUPP: ignoring macro in table: BR bmi
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1960:2: UNSUPP: ignoring macro in table: BR clip
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1963:2: UNSUPP: ignoring macro in table: BR
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1966:2: UNSUPP: ignoring macro in table: BR cr
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1969:2: UNSUPP: ignoring macro in table: BR da
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1972:2: UNSUPP: ignoring macro in table: BR dr
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1975:2: UNSUPP: ignoring macro in table: BR	fdups
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1978:2: UNSUPP: ignoring macro in table: BR +
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1984:2: UNSUPP: ignoring macro in table: BR dh
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1987:2: UNSUPP: ignoring macro in table: BR h
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1993:2: UNSUPP: ignoring macro in table: BR *
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1996:2: UNSUPP: ignoring macro in table: BR **
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:1999:2: UNSUPP: ignoring macro in table: BR
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2002:2: UNSUPP: ignoring macro in table: BR ih
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2005:2: UNSUPP: ignoring macro in table: BR i
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2008:2: UNSUPP: ignoring macro in table: BR ++
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2011:2: UNSUPP: ignoring macro in table: BR kbgen
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2014:2: UNSUPP: ignoring macro in table: BR kd
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2017:2: UNSUPP: ignoring macro in table: BR ke
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2020:2: UNSUPP: ignoring macro in table: BR ml
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2023:2: UNSUPP: ignoring macro in table: BR music
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2026:2: UNSUPP: ignoring macro in table: BR gg
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2029:2: UNSUPP: ignoring macro in table: BR ptot
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2032:2: UNSUPP: ignoring macro in table: BR	rrm
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2035:2: UNSUPP: ignoring macro in table: BR //
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2038:2: UNSUPP: ignoring macro in table: BR
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2041:2: UNSUPP: ignoring macro in table: BR vid
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2044:2: UNSUPP: ignoring macro in table: BR	vt
mandoc: /Volumes/Storage/Developer/clifm/misc/./manpage:2047:2: UNSUPP: ignoring macro in table: BR wall
```

**Expected behavior**
No warnings/errors should be printed to the terminal when viewing the manpage.

**Desktop (please complete the following information):**
 - OS: macOS 12.6.1 (21G217)
 - Terminal: iTerm2 (xterm-256color)
 - CliFM version: 1.9
- Installation source: Git

**Additional context**
`man --version` prints `man, version 1.6g`

**Comment by leo-arch** [comment]
Thanks for reporting @alebcay. I'll take a look at it and let you know.

Btw, how did you compile **clifm** on macOS? If your compilation instructions work fine I would like to add them to the docs.

**Comment by leo-arch** [comment]
Please give it a try now. The offending macro was removed.

**Comment by alebcay** [comment]
Confirmed the warning/error messages are no longer showing up when displaying the manpage from latest `HEAD`.

Regarding the build process, in my case I am using `libmagic`, `readline`, and `gettext` installed from Homebrew. To avoid conflicts with the macOS bundled copy of `libedit`, Homebrew does not automatically symlink `reaadline` into a location that `pkg-config` can find it. Thus, I needed to `export PKG_CONFIG_PATH="/usr/local/opt/readline/lib/pkgconfig"` (this path may differ between Intel vs ARM macOS or if the user has installed Homebrew somewhere else).

The process is probably similar if using MacPorts or some other package manager to get the dependencies - just need to make sure they're all included in the pkg-config search path.

After sorting out where `readline` is located, I just `cmake -S . -B build [...] && cmake --build build && cmake --install build`.
